resources/components/SongSearchCard.jsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import Typography from "@mui/material/Typography";
+
+export default function SongSearchCard({ song }) {
+    return (
+        <Card sx={{ maxWidth: 150 }}>
+            <CardMedia
+                component="img"
+                alt="green iguana"
+                height="140"
+                image={song.song_data.album_cover_img}
+            />
+            <CardContent>
+                <Typography gutterBottom variant="body1" component="h2">
+                    {song.song_data.song_name.length >= 14
```

Comment:
- Question: Pourquoi n'as tu pas essayé d'utiliser text-ellipsis en css ? 
- Sinon , si tu décide de garder cette logique, ça pourrait être cool d'avoir une methode dans "helpers/String.js"  qui s'appellerai "textElipsis" (car tu as de la logique répétée)

Reply:
found an equivalen in material UI: `noWrap`
Taking into account that there is a fix width of a card container, noWrap works well: <Typography gutterBottom variant="body1" component="h2" noWrap>
                    {song.song_data.song_name}
                </Typography>